treeherder/log_parser/tasks.py

```diff
@@ -57,5 +58,8 @@ def store_error_summary(project, job_log_url, job_guid):
     try:
         logger.info('Running store_error_summary')
```

Comment:
We log .info() by default on stage/prod, so perhaps this should be .debug() instead to reduce log noise?